jenkins/development.Dockerfile

```diff
@@ -0,0 +1,113 @@
+# Fedora 33 pbench development image
```

Comment:
Not sure where, but we really need to document somewhere that using the `black` RPM provided by Fedora 33 and later does not work for our code base and we have to use a `pip` installed version explicitly.

Reply:
I enlarged the commentary in the Dockerfile.  Ideally, we would periodically review the test console output to ensure that it reports nothing surprising.